<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eggsizerML</class>
 <widget class="QMainWindow" name="eggsizerML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eggsizerML</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Select + Display</string>
    </property>
    <widget class="QPushButton" name="fileOpen_btn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Uploaded Image:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgDisp_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>321</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>170</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Processed Image:</string>
     </property>
    </widget>
    <widget class="QLabel" name="imgDisp_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>321</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="value">
      <number>33</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Sigma Value:</string>
     </property>
    </widget>
    <widget class="QLabel" name="sigma_val_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
